bot/exts/moderation/clean.py

```diff
@@ -179,7 +179,7 @@ def predicate_regex(message: Message) -> bool:
 
         def predicate_range(message: Message) -> bool:
             """Check if the message age is between the two limits."""
-            return first_limit <= message.created_at <= second_limit
+            return first_limit < message.created_at < second_limit
```

Comment:
I'm confused how this makes a difference. The messages are fetched using `Messageable.history()` with the same limits. For `history()`, those limits are already exclusive. I don't understand why `predicate_range` and `predicate_after` are even needed.

Reply:
Because the search might go through the cache. If history is indeed exclusive then I guess this predicate could be dropped in those cases, but I don't mind another guarantee if it already exists.

Reply:
`predicate_after` should also be exclusive then.

Reply:
Hmm good point. It doesn't cause a bug because the iteration excludes the stopping point.